src/AuthContext.tsx

```diff
@@ -50,10 +50,10 @@ export const initUserManager = (props: AuthProviderProps): UserManager => {
     popupWindowTarget,
   } = props;
   return new UserManager({
-    authority,
-    client_id: clientId,
+    authority: authority || '',
+    client_id: clientId || '',
     client_secret: clientSecret,
-    redirect_uri: redirectUri,
+    redirect_uri: redirectUri || '',
```

Comment:
Why are these being defaulted to an empty string? It's been a while since I worked with this code base, so I'm a bit rusty 😅